app/models/project_role.rb

```diff
@@ -5,6 +5,7 @@ class ProjectRole < ActiveHash::Base
   include ActiveModel::Serializers::JSON
 
   self.data = [
+    { id: -1, name: "default" },
```

Comment:
Hadn't noticed that @henders had already commented on this. I guess `viewer` gets the most votes. :smile:.